docker-compose.yaml

```diff
@@ -9,9 +9,17 @@ services:
     secrets:
       - service_account_key
     volumes:
-      - "./app:/app/app"
       - "./junit:/tmp/junit"
 
 secrets:
   service_account_key:
     file: ./service-account.json
+######################
+# Debugging
+######################
+
+# entrypoint: /bin/bash -l -c "tail -f /dev/null"
+# command: tail -f /dev/null
+# volumes:
+#    - "./app:/app"
+#    - "./service-account.json:/app/service-account.json"
```

Comment:
I'd prefer not to have commented directives. I recommend storing this in a `docker-compose.override.yml` file (which should be git ignored), if these are personal preferences. If they are intended to be used by others, I suggest creating a separate docker compose file - e.g. `docker-compose.debug.yml`.

Reply:
All good suggestions, I'll move over to troubleshooting in the readme